test/util/e2e.go

```diff
@@ -308,3 +309,19 @@ func GetKuberayTestImage() string {
 	gomega.Expect(found).To(gomega.BeTrue())
 	return kuberayTestImage
 }
+
+func CreateNamespaceWithLog(ctx context.Context, k8sClient client.Client, nsName string) *corev1.Namespace {
+	ginkgo.GinkgoHelper()
+	ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: nsName}}
+	gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
+	ginkgo.By(fmt.Sprintf("Created namespace: %s", ns.Name))
+	return ns
+}
+
+func CreateNamespaceFromPrefixWithLog(ctx context.Context, k8sClient client.Client, nsPrefix string) *corev1.Namespace {
+	ginkgo.GinkgoHelper()
+	ns := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{GenerateName: nsPrefix}}
+	gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
+	ginkgo.By(fmt.Sprintf("Created namespace: %s (from prefix %s)", ns.Name, nsPrefix))
```

Comment:
```suggestion
	ginkgo.By(fmt.Sprintf("Created namespace: %s", ns.Name))
```
I think logging the passed prefix feels like too much detail TBH

Reply:
sure